src/nanoemoji/colr_to_svg.py

```diff
@@ -63,9 +67,8 @@ def _map_font_space_to_viewbox(
 
 
 def _svg_root(view_box: Rect) -> etree.Element:
-    svg_tree = etree.parse(
-        str(test_helper.locate_test_file("colr_to_svg_template.svg"))
-    )
+    with resources.path("nanoemoji.data", "colr_to_svg_template.svg") as template_file:
```

Comment:
it seems a bit overkill to use importlib.resources to embed this file, it's very small and you could simply write inline as a string literal inside the code somewhere.

```python
COLR_TO_SVG_TEMPLATE = """\
<svg viewBox="TBD" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <defs/>
</svg>"""
```

Reply:
Is importlib considered "heavy"? 

(no objection to the proposed change, just curious)

Reply:
no, not heavy per se, it just feels to me a bit overengineering for such a short strings. E.g. for the default config.toml, it makes more sense. Anyway I don't mind if you keep it.

Reply:
I killed it